content/reference/user/reports/index.md

```diff
@@ -199,6 +210,9 @@ In the top right menu, you can click on the **Percentiles** setting to be able t
 
 In the top right menu, you can activate the **Date Time** setting to be able to switch from elapsed time to date time.
 
+{{< img src="multiplecheck-date.png" alt="Multiple check" >}}
+{{< img src="multiple-date.png" alt="Multiple" >}}
+
 #### Highlight Legend
 
 By hovering the label on the percentiles chart legend, you will be able to highlight the curve on the chart, leading to a better view of that curve.
```

Comment:
```suggestion
If you hover your mouse over a label on the percentiles chart legend, you will highlight the curve on the chart, leading to a better view of that curve.
```

Reply:
Also, line 219 : "The highlight legend option is enabled for every "non stacked" graph."